src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -436,6 +439,10 @@ class boss_vazruden_the_herald : public CreatureScript
                                 return;
                             }
                         }
+            if (!(Nazan && Nazan->IsAlive()) && !(Vazruden && Vazruden->IsAlive()))
+            {
+                me->DisappearAndDie();
+            }
```

Comment:
here needs fix indent.

Reply:
Thanks. I forgot to compare with the previous indents above.